### ISSUE TITLE
OTA does not work

### ISSUE THREAD

**Issue by Roycinger** [NimmLor/esp8266-fastled-iot-webserver#171]
Hi,
although I configured the OTA module parameters, the upload does not work.
I'm always getting prompted for the OTA password (although it's set).
Additionally, I get a "No response from device" message.
![image](https://user-images.githubusercontent.com/7791335/109966533-aceadb00-7cf0-11eb-8b23-7432b51772c8.png)

Does anyone deal with a similar problem?
Thanks!

**Comment by bb-Ricardo** [comment]
Does it work without password?

**Comment by Roycinger** [comment]
> Does it work without password?

Unfortuantely not

**Comment by bb-Ricardo** [comment]
Did you start the OTA mode via Web interface?

**Comment by Roycinger** [comment]
> Did you start the OTA mode via Web interface?

Yes. The LEDs freeze, UI unavailable (as per design), the ESP gets displayed in the Arduino IDE under ports.
As soon as the sketch is compiled and the actual upload begins, I'm getting the described behaviour

**Comment by bb-Ricardo** [comment]
I'm sorry. I actually don't know how to debug this. What do you get in your serial console when you try to perform a OTA Update?

**Comment by Roycinger** [comment]
> I'm sorry. I actually don't know how to debug this. What do you get in your serial console when you try to perform a OTA Update?

Sorry, I didnt see the last notification.
Serial Monitor is not available when doing an OTA update

**Comment by obenschlaefer** [comment]
<img width="749" alt="image" src="https://github.com/NimmLor/esp8266-fastled-iot-webserver/assets/79227566/74dba1e7-1bca-4336-a8f4-5cd10576b4ab">
 
 you can find a password option **in line 151** (maybe +-5 lines in your code) 
 
 Just try to find it by using the search option in the arduino IDE (strg+f)
